include/alpaka/vec/Vec.hpp

```diff
@@ -368,6 +368,12 @@ namespace alpaka
             return (*this)[I];
         }
 
+        //! \return The number of dimensions of the vector.
+        [[nodiscard]] ALPAKA_FN_HOST_ACC static consteval auto dim() noexcept -> TVal
```

Comment:
Didn't we agree on `dims` (or similar) here?

Reply:
We agreed with the Simeon's suggession, dim.

Reply:
I talked with @psychocoderHPC . Because it returns only a single number and `std::mdspan` use `rank()` and not `ranks()` , singular makes more sense.